test/test.js

```diff
@@ -9,7 +9,7 @@ describe("loader", function() {
 			query: "?abc.def.ghi=>1"
 		}, "").should.be.eql(HEADER +
 			"var abc = (abc || {});\n" +
-			"abc.def = {};\n" +
+			"abc.def = abc.def || {};\n" +
```

Comment:
This is my fault for not paying attention when I looked at it originally but as a life rule in JavaScript, don't use eval. Even when it isn't a gigantic gaping security issue, there is always a way to do the same thing without it.